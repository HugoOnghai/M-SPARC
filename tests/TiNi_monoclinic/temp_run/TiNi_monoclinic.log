
                            < M A T L A B (R) >
                  Copyright 1984-2019 The MathWorks, Inc.
                  R2019a (9.6.0.1072779) 64-bit (glnxa64)
                               March 8, 2019

 
To get started, type doc.
For product information, visit www.mathworks.com.
 

 Reading .inpt file ...


<INPT>
# $ cat TiNi_monoclinic.inpt
# nprocs: 24

# Test: TiNi
LATVEC:
0.981853167430541   0.000000000000   -0.189642710396718
0.000000000000000   1.00000000000000  0.000000
0.000000000000000   0.00000000000000  1.000000000000000
CELL: 7.662855150000000   5.508562950000000   8.983776419999998
MESH_SPACING: 0.20
FD_ORDER: 12
BC: P P P
KPOINT_GRID: 1 1 1
EXCHANGE_CORRELATION: GGA_PBE
ELEC_TEMP_TYPE: fermi-dirac
ELEC_TEMP: 315.775131
TOL_SCF: 1e-6
CALC_PRES: 1
CALC_STRESS: 1
PRINT_FORCES: 1
PRINT_ATOMS: 0
MIXING_VARIABLE: potential
MIXING_PRECOND: none
<\INPT>

 Reading .ion file ...
 Number of atom types : 2
 Total number of atoms: 4
 Default atomic mass for Ti is 47.867000
 Default atomic mass for Ni is 58.693400
 pseudo_dir # 1 = Ti.psp8
 pseudo_dir # 2 = Ni.psp8

 COORD:
   4.632065669637450   4.131422212500000   2.557321795717200
   3.030789480362550   1.377140737500000   6.426454624282798
   7.228746742897350   4.131422212500000   7.438791470170498
   0.434108407102650   1.377140737500000   1.544984949829500

  40.866403969098997

     1.447538698402000e+02

 kpoint grid before symmetry:
     0     0     0

 kpoint grid after symmetry:
     0     0     0

## Chebyshev polynomial degree not provided, finding npl ...
## Based on the mesh size, npl is set to: 35
## Number of states not provided, finding Nev ...
## Based on the number of electrons, Nev is set to: 41
## Poisson tolerance not provided, choosing poisson_tol ...
## poisson_tol is set to: 1.000e-08
## Pseudocharge tolerance not provided, choosing pseudocharge_tol ...
## pseudocharge_tol is set to: 1.000e-08
 Creating differentiation matrices ...
 Done. (2.581 sec)

 Estimated memory usage:
 Total:  193.61 MB
 orbitals            :   59.91 MB
 sparse matrices     :  123.47 MB
 global-size vectors :    5.11 MB
 mixing histories    :    5.11 MB

 Finding rb for Ti ...
 rb = {2.015 1.996 1.996}, int_b = -12.000146869915737, err_rb = 1.469e-04
 rb = {2.214 2.196 2.196}, int_b = -12.000121224263555, err_rb = 1.212e-04
 rb = {2.414 2.396 2.396}, int_b = -12.000090098310531, err_rb = 9.010e-05
 rb = {2.613 2.595 2.595}, int_b = -12.000056734060397, err_rb = 5.673e-05
 rb = {2.813 2.795 2.795}, int_b = -12.000040693323346, err_rb = 4.069e-05
 rb = {3.013 2.995 2.995}, int_b = -12.000028016152864, err_rb = 2.802e-05
 rb = {3.212 3.194 3.194}, int_b = -12.000019923158838, err_rb = 1.992e-05
 rb = {3.412 3.394 3.394}, int_b = -12.000012929385766, err_rb = 1.293e-05
 rb = {3.612 3.594 3.594}, int_b = -12.000006554012005, err_rb = 6.554e-06
 rb = {3.811 3.793 3.793}, int_b = -12.000001021572123, err_rb = 1.022e-06
 rb = {4.011 3.993 3.993}, int_b = -11.999995604293137, err_rb = 4.396e-06
 rb = {4.211 4.192 4.192}, int_b = -11.999991844507036, err_rb = 8.155e-06
 rb = {4.410 4.392 4.392}, int_b = -11.999988302595636, err_rb = 1.170e-05
 rb = {4.610 4.592 4.592}, int_b = -11.999987075025382, err_rb = 1.292e-05
 rb = {4.809 4.791 4.791}, int_b = -11.999986232198571, err_rb = 1.377e-05
 rb = {5.009 4.991 4.991}, int_b = -11.999986994966834, err_rb = 1.301e-05
 rb = {5.209 5.191 5.191}, int_b = -11.999988037458943, err_rb = 1.196e-05
 rb = {5.408 5.390 5.390}, int_b = -11.999989401997905, err_rb = 1.060e-05
 rb = {5.608 5.590 5.590}, int_b = -11.999991093633934, err_rb = 8.906e-06
 rb = {5.808 5.790 5.790}, int_b = -11.999991772112351, err_rb = 8.228e-06
 rb = {6.007 5.989 5.989}, int_b = -11.999994045853445, err_rb = 5.954e-06
 rb = {6.207 6.189 6.189}, int_b = -11.999998472826954, err_rb = 1.527e-06
 rb = {6.407 6.388 6.388}, int_b = -12.000000162291544, err_rb = 1.623e-07
 rb = {6.606 6.588 6.588}, int_b = -11.999999965650417, err_rb = 3.435e-08
 rb = {6.806 6.788 6.788}, int_b = -12.000000007802178, err_rb = 7.802e-09
 rb = {6.806 6.788 6.788}
 Finding rb for Ni ...
 rb = {2.015 1.996 1.996}, int_b = -18.000316119976368, err_rb = 3.161e-04
 rb = {2.214 2.196 2.196}, int_b = -18.000075265773276, err_rb = 7.527e-05
 rb = {2.414 2.396 2.396}, int_b = -18.000082639753607, err_rb = 8.264e-05
 rb = {2.613 2.595 2.595}, int_b = -18.000060723215348, err_rb = 6.072e-05
 rb = {2.813 2.795 2.795}, int_b = -18.000041441042736, err_rb = 4.144e-05
 rb = {3.013 2.995 2.995}, int_b = -18.000027752542515, err_rb = 2.775e-05
 rb = {3.212 3.194 3.194}, int_b = -18.000017862451912, err_rb = 1.786e-05
 rb = {3.412 3.394 3.394}, int_b = -18.000011061754904, err_rb = 1.106e-05
 rb = {3.612 3.594 3.594}, int_b = -18.000006632533317, err_rb = 6.633e-06
 rb = {3.811 3.793 3.793}, int_b = -18.000004242419571, err_rb = 4.242e-06
 rb = {4.011 3.993 3.993}, int_b = -18.000002463476992, err_rb = 2.463e-06
 rb = {4.211 4.192 4.192}, int_b = -18.000001935041478, err_rb = 1.935e-06
 rb = {4.410 4.392 4.392}, int_b = -18.000000757264186, err_rb = 7.573e-07
 rb = {4.610 4.592 4.592}, int_b = -18.000000821069861, err_rb = 8.211e-07
 rb = {4.809 4.791 4.791}, int_b = -18.000000329479025, err_rb = 3.295e-07
 rb = {5.009 4.991 4.991}, int_b = -18.000000303271065, err_rb = 3.033e-07
 rb = {5.209 5.191 5.191}, int_b = -17.999999772696405, err_rb = 2.273e-07
 rb = {5.408 5.390 5.390}, int_b = -17.999998962421291, err_rb = 1.038e-06
 rb = {5.608 5.590 5.590}, int_b = -17.999998109249901, err_rb = 1.891e-06
 rb = {5.808 5.790 5.790}, int_b = -17.999996288822150, err_rb = 3.711e-06
 rb = {6.007 5.989 5.989}, int_b = -17.999996045918312, err_rb = 3.954e-06
 rb = {6.207 6.189 6.189}, int_b = -17.999998677441802, err_rb = 1.323e-06
 rb = {6.407 6.388 6.388}, int_b = -18.000000150036431, err_rb = 1.500e-07
 rb = {6.606 6.588 6.588}, int_b = -17.999999968132567, err_rb = 3.187e-08
 rb = {6.806 6.788 6.788}, int_b = -18.000000006977050, err_rb = 6.977e-09
 rb = {6.806 6.788 6.788}

 ###############################################################
 Relaxation step number: 1 

 Starting pseudocharge generation and self energy calculation...
 Integration b = 60.000000008699

 Integration b_ref = 60.000000000070

 ****************************************
 *          Eself_ref = 1671.196208       *
 ****************************************
 Done. (3.754282 s)
 Time for b calculation: 3.768 seconds.

 Starting calculating nonlocal projectors ...
 Done. (2.257761 s)

 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ 
 Starting SCF iteration...
 AAR converged to a relative residual of 6.71463e-09 in 85 iterations.
 Poisson problem took 2.858095s
 ============================================= 
 Relaxation iteration:  1
 SCF iteration number:  1, Chebyshev cycle: 1 
 ============================================= 
 Fermi energy = 1.179017
 This SCF iteration took 9.418 s.

 ============================================= 
 Relaxation iteration:  1
 SCF iteration number:  1, Chebyshev cycle: 2 
 ============================================= 
 Fermi energy = -2.374292
 This SCF iteration took 7.703 s.

 ============================================= 
 Relaxation iteration:  1
 SCF iteration number:  1, Chebyshev cycle: 3 
 ============================================= 
 Fermi energy = -2.528555
 This SCF iteration took 7.848 s.

 ============================================= 
 Relaxation iteration:  1
 SCF iteration number:  1, Chebyshev cycle: 4 
 ============================================= 
 Fermi energy = -2.539920
 AAR converged to a relative residual of 4.11921e-09 in 91 iterations.
 Poisson problem took 2.989573s
 Escc = 1030.05414948
 Etot = -221.15817262
 Eatom = -55.28954315
 Error in SCF iteration: 1.8816e+00 
 This SCF iteration took 10.749 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number:  2 
 ========================= 
 Fermi energy = 0.823809
 AAR converged to a relative residual of 6.42503e-09 in 91 iterations.
 Poisson problem took 2.983941s
 Escc = 392.44560918
 Etot = -385.31991641
 Eatom = -96.32997910
 Error in SCF iteration: 1.6677e+00 
 This SCF iteration took 10.759 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number:  3 
 ========================= 
 Fermi energy = 0.830543
 AAR converged to a relative residual of 4.73895e-09 in 85 iterations.
 Poisson problem took 2.773083s
 Escc = 76.59163923
 Etot = -432.12446213
 Eatom = -108.03111553
 Error in SCF iteration: 4.5403e-01 
 This SCF iteration took 10.587 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number:  4 
 ========================= 
 Fermi energy = 0.717097
 AAR converged to a relative residual of 3.75547e-09 in 91 iterations.
 Poisson problem took 2.961472s
 Escc = 19.17662630
 Etot = -371.83985223
 Eatom = -92.95996306
 Error in SCF iteration: 9.9814e-01 
 This SCF iteration took 10.759 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number:  5 
 ========================= 
 Fermi energy = 0.197285
 AAR converged to a relative residual of 6.21844e-09 in 85 iterations.
 Poisson problem took 2.782826s
 Escc = 42.37903974
 Etot = -405.17165048
 Eatom = -101.29291262
 Error in SCF iteration: 8.4427e-01 
 This SCF iteration took 10.534 s.

 ========================= 
 Relaxation iteration:  1 
 SCF iteration number:  6 
 ========================= 
