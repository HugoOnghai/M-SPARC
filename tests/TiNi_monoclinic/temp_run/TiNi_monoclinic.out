***************************************************************************
*                          M-SPARC (Feb 20, 2020)                         *
*   Copyright (c) 2019 Material Physics & Mechanics Group, Georgia Tech   *
*           Distributed under GNU General Public License 3 (GPL)          *
*                Date: 11-May-2021  Start time: 21:05:34                  *
***************************************************************************
                           Input parameters                                
***************************************************************************
CELL: 7.662855 5.508563 8.983776 
LATVEC:
0.981853167430541 0.000000000000000 -0.189642710396718 
0.000000000000000 1.000000000000000 0.000000000000000 
0.000000000000000 0.000000000000000 1.000000000000000 
FD_GRID: 38 28 45
FD_ORDER: 12
BC: P P P
KPOINT_GRID: 1 1 1
KPOINT_SHIFT: 0 0 0
ELEC_TEMP_TYPE: fermi-dirac
SMEARING: 0.001000000
CHEB_DEGREE: 35
NSTATES: 41
EXCHANGE_CORRELATION: GGA_PBE
CALC_STRESS: 1
MAXIT_SCF: 100
TOL_SCF: 1.00E-06
TOL_POISSON: 1.00E-08
TOL_LANCZOS: 1.00E-02
TOL_PSEUDOCHARGE: 1.00E-08
MIXING_VARIABLE: potential
MIXING_PRECOND: none
MIXING_PARAMETER: 0.30
MIXING_HISTORY: 7
PULAY_FREQUENCY: 1
PULAY_RESTART_FREQ: 0
REFERENCE_CUTOFF: 0.50
RHO_TRIGGER: 4
FIX_RAND: 0
PRINT_FORCES: 1
PRINT_ATOMS: 0
PRINT_EIGEN: 0
PRINT_DENSITY: 0
OUTPUT_FILE: TiNi_monoclinic.out
***************************************************************************
                             Initialization                                
***************************************************************************
Mesh spacing in x-direction        :  0.201654 (Bohr)
Mesh spacing in y-direction        :  0.196734 (Bohr)
Mesh spacing in z direction        :  0.199639 (Bohr)
Number of symmetry adapted k-points:  1
Output printed to                  :  TiNi_monoclinic.out
Total number of atom types         :  2
Total number of atoms              :  4
Total number of electrons          :  60
Atom type 1  (valence electrons)   :  Ti 12
Pseudopotential                    :  Ti.psp8
lloc                               :  4
Pseudocharge radii of atom type 1  :  6.81 6.79 6.79
Number of atoms of type 1          :  2
Atom type 2  (valence electrons)   :  Ni 18
Pseudopotential                    :  Ni.psp8
lloc                               :  4
Pseudocharge radii of atom type 2  :  6.81 6.79 6.79
Number of atoms of type 2          :  2
Estimated total memory usage       :  193.61 MB
=====================================================================
                    Self Consistent Field (SCF#1)                     
=====================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -5.5289543154E+01        1.882E+00        35.716
2            -9.6329979103E+01        1.668E+00        10.757
3            -1.0803111553E+02        4.540E-01        10.585
4            -9.2959963057E+01        9.981E-01        10.757
5            -1.0129291262E+02        8.443E-01        10.532
